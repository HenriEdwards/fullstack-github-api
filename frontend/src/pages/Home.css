* {
  padding: 5px;
}

.search-form{
  background-color:#347D39;
  padding: 25px;
  margin-top: 25px;
  border-radius: 10px;
  color: white;

}

.user{
  padding: 15px;
  color: #ACBAC7;
}

.link {
  color: #28a745;
  padding-top: 15px;
}
.user-name{
  padding-bottom: 15px;
}

.search-text{
  padding-bottom: 15px;
}

button{
  border-radius: 5px;
  padding: 5px;
  color:rgb(233, 233, 233);
  font-weight: 800;
  background-color: #28a745;
}

button:hover{
  background-color: #1f8336;
}

input{
  border-radius: 5px;
  padding: 5px;
  color: #2b2b2b;
  font-weight: 800;
  background-color: rgb(233, 233, 233);
}