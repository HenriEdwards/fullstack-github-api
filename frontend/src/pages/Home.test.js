import React from "react";
import { render } from "@testing-library/react";
import Home from "./Home";

test("renders Home component correctly", () => {
  // Render Home component
  const { container } = render(<Home />);

  // Compare rendered output with the existing snapshot
  expect(container).toMatchSnapshot();
});
