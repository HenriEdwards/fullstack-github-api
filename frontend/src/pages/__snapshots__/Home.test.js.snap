// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="search-form"
    >
      <h4
        class="search-text"
      >
        Search GitHub by Username
      </h4>
      <form>
        <input
          type="text"
          value=""
        />
        <button
          type="submit"
        >
          Search
        </button>
      </form>
    </div>
    <div
      class="search-results"
    />
  </div>
</div>
`;
